sdcm/logcollector.py

```diff
@@ -1134,6 +1136,34 @@ def is_collect_to_a_single_archive(self) -> bool:
         return True
 
 
+class SSTablesCollector(SCTLogCollector):
+    """
+    Collect corrupted sstables from db nodes.
+    """
+    log_entities = [
+        FileLog(name='parallel-timelines-report.html',
+                search_locally=True)
+    ]
+    cluster_log_type = "corrupted-sstables"
+    cluster_dir_prefix = "corrupted-sstables"
```

Comment:
why is it duplicated? couldn't it be something like:
`cluster_log_type = cluster_dir_prefix = "corrupted-sstables"` ?

Reply:
Just taken from other log collector classes - actually not used and possibly can be removed.

Reply:
but it wasn't removed in the last changes, or was re-introduced in last changes

Reply:
Wasn't removed. I forgot about this, but anyway I don't see it as problem as all other log collectors use it in the same way.